workflow/rules/bwa_mem.smk

```diff
@@ -19,10 +19,17 @@ rule bwa_mem:
         "aligment/bwa_mem/{sample}_{unit}.bam.log",
     benchmark:
         "alignment/bwa_mem/{sample}_{unit}.bam.benchmark.tsv"
+    benchmark:
+        repeat(
+            "aligment/bwa_mem/{sample}_{unit}.output.bam.benchmark.tsv",
+            config.get("rule_name", {}).get("benchmark_repeats", 1),
+        )
     threads: config["bwa_mem"]["threads"]
     container:
-        config["bwa_mem"]["container"]
+        config.get("bwa_mem", "default_container").get("container", "default_container")
+    conda:
+        "../envs/rule_name.yaml"
```

Comment:
Again, it should be `bwa_mem`, right?

Reply:
It actually workes with:
"../envs/{rule}.yaml"